packages/core/src/Stores/client-store.js

```diff
@@ -384,9 +383,10 @@ export default class ClientStore extends BaseStore {
         const { gaming_company, financial_company } = this.landing_companies;
         const financial_shortcode = financial_company?.shortcode;
         const gaming_shortcode = gaming_company?.shortcode;
-        return financial_shortcode || gaming_shortcode
-            ? eu_shortcode_regex.test(financial_shortcode) || eu_shortcode_regex.test(gaming_shortcode)
-            : eu_excluded_regex.test(this.residence);
+        return (
+            ((financial_shortcode || gaming_shortcode) && eu_shortcode_regex.test(financial_shortcode)) ||
+            eu_shortcode_regex.test(gaming_shortcode)
```

Comment:
I think your brackets are wrong? shouldn't it be `((financial_shortcode || gaming_shortcode) && (eu_shortcode_regex.test(financial_shortcode) ||
            eu_shortcode_regex.test(gaming_shortcode))`
also why are we removing the fallback to checking residence?